<launch>
    <group ns="tf03_driver">
        <!-- Current parameters -->
        <rosparam param="sensor_frame"> ["front"]</rosparam>
        <rosparam param="sensor_interface">serial</rosparam>
        <rosparam param="serial_port">/dev/ttyUSB0</rosparam>

        <!-- Desired parameters -->
        <!-- Make sure to set all parametres so you dont end up with default values -->>
        <rosparam param="set_transmit_can_id"> 1</rosparam>
        <rosparam param="set_receive_can_id"> 17</rosparam>
        <rosparam param="set_output_format">can</rosparam>        
        <rosparam param="print_version">false</rosparam>
    </group>
    <node pkg="tf03_driver" name="tf03_driver" type="tf03_configuration_node" output="screen" required="true">
    </node>

</launch>
